src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
+
+* FEATURE: Add new PE CV_CFL_LANG language code for ALIASOBJ and Rust. [530](https://github.com/microsoft/binskim/pull/530)
```

Comment:
>CV_CFL_LANG [](http://example.com/codeflow?start=22&length=11)

This file does not do a good job of using markdown conventions of properly denoting code items, for example, by encapsulating them in backticks.

Reply:
added backticks.